waku/v2/protocol/store/message_queue.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+type MessageQueue struct {
```

Comment:
the message queue doesn t really deal with retention time? only the number of message we store.As far as I understand, the queue is here as a cache layer?

Reply:
Yeah, currently it only deals with number of messages. Perhaps the goroutine for cleaning older records can be added here?